docs/pages/flexbox-mode.md

```diff
@@ -13,13 +13,14 @@ Foundation components use a combination of floats, vertical alignment, table cel
 
 Enabling **flexbox mode** replaces those hacks with flexbox properties, streamlining how layouts are made, and making sizing and alignment of elements much easier.
 
-Flexbox mode is only supported these browsers:
+Flexbox mode is only supported in these browsers:
 
 - The latest Chrome and Firefox
 - Safari 6+
 - IE 10+
 - iOS 7+
 - Android 4.4+
+- Edge 12+
```

Comment:
As Edge versions follow IE versions, and because Edge is nothing but Edge with a new name, I think the following would be more appropriate:

> - IE/Edge 10+ 

This is often this way we talk about IE and Edge compatibility in others frameworks.